main.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, render_template, request, redirect, url_for
+import pandas as pd
+import json
+import plotly
+import plotly.express as px
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+ 
+app = Flask(__name__)
+
+df = pd.read_csv('women_EU2.csv', sep=',')
+
+@app.route('/', methods=['GET','POST'])
+def search():
+	df = pd.read_csv('women_EU2.csv', sep=',')
+	if request.method == 'POST':
+		country_name = request.form.get('country')
+		return redirect(f"/{country_name}")
+		#return redirect(url_for("country_plot", country=country_name))
+	return render_template('base.html', table=df)
+
+
+
+@app.route('/<country>')
+def country_plot(country):
```

Comment:
Would it be possible to add comments above each line below explaining what is happening? 

Eg:
```
# Load data from file
df = pd.read_csv('women_EU2.csv', sep=',')

# Get data for selected country
df_country = df[df['country'] == country]
```